yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -493,7 +501,7 @@ func (f *ForeignTableIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]Q
 	for _, col := range foreignTable.Columns {
 		isUnsupportedDatatype := utils.ContainsAnyStringFromSlice(srcdb.PostgresUnsupportedDataTypes, col.TypeName)
 		if isUnsupportedDatatype {
-			reportUnsupportedDatatypes(col, obj.GetObjectType(), foreignTable.GetObjectName(), &issues)
+			issues = append(issues, ReportUnsupportedDatatypes(col.TypeName, col.ColumnName, obj.GetObjectType(), foreignTable.GetObjectName()))
```

Comment:
didn't get this.
What is the difference here in before vs after.
Was passing reference to issues slice had any problem?

Reply:
This was done so that I could re-use this function in assess migration. I wanted to return the queryissue instead of directly appending it to the list in the function. This returned queryissue could be then converted to Assessment Issue.